src/glyph/serialize.rs

```diff
@@ -57,8 +60,11 @@ impl Glyph {
         start.push_attribute(("format", "2"));
         writer.write_event(Event::Start(start)).map_err(GlifWriteError::Xml)?;
 
+        let mut seen_codepoints = HashSet::new();
```

Comment:
maybe we just enforce deduplication whenever the user modifies the codepoints? As in we always keep them sorted, and we don't insert dupes (we can use `slice::binary_search` to find the insertion point, or whether the item already exists)

Reply:
Hm... see my note above, not sure if it's just easier to do it at read/write time?